### ISSUE TITLE
Error: Cannot find module './lib/MongoDbStorage'

### ISSUE THREAD

**Issue by kostyaVyrodov** [botbuilder-contrib/botbuilder-storage-mongodb#30]
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
1. Install `botbuilder-storage-mongodb`
1. Import the package
1. Run your app

### Any log messages given by the failure
> Error: Cannot find module './lib/MongoDbStorage'

```Require stack:
- /Users/kvyro/Projects/<app-name>/node_modules/botbuilder-storage-mongodb/index.js
- /Users/kvyro/Projects/<app-name>/src/app.ts
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/Users/kvyro/Projects/<app-name>/node_modules/botbuilder-storage-mongodb/index.js:1:22)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
```

### Expected/desired behavior
The app should be able to start

### OS and Version?
MacOS Catalina v10.15.3

### Version(s)
1.0.4

### Mention any other details that might be useful

The built npm package in my node_modules looks like on the screenshot below

![image](https://user-images.githubusercontent.com/15310397/92494124-fa810200-f1fd-11ea-8f2d-651b7b617aaf.png)

**Comment by nurih** [comment]
Looks like the original `index.js` incorrectly refers to the subfolder `lib`.

**Comment by kostyaVyrodov** [comment]
@nurih, yes. But I don't see the `lib` folder in the package. Should it be there?

**Comment by nurih** [comment]
@kostyaVyrodov the bug is that `index.js` is incorrect. It should not have included the subfolder `lib` . The fix (not yet published) in #31 will introduce index.js into the NPM package that correctly refers to the storage module.

**Comment by nurih** [comment]
Release [v1.0.5](https://www.npmjs.com/package/botbuilder-storage-mongodb/v/1.0.5) published to fix your issue.

Thank you for reporting this!
